Makefile

```diff
@@ -60,9 +60,9 @@ export KUBECTL ?= ./cluster/kubectl.sh
 KUBECTL ?= ./cluster/kubectl.sh
 OPERATOR_SDK ?= $(GOBIN)/operator-sdk
 
-GINKGO = go run github.com/onsi/ginkgo/v2/ginkgo
-CONTROLLER_GEN = go run sigs.k8s.io/controller-tools/cmd/controller-gen
-OPM = go run -tags=json1 github.com/operator-framework/operator-registry/cmd/opm
+GINKGO = GOFLAGS=-mod=mod go run github.com/onsi/ginkgo/v2/ginkgo@v2.1.4
+CONTROLLER_GEN = GOFLAGS=-mod=mod go run sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.0
```

Comment:
Could we have something like 

`GO_MOD_FLAGS = GOFLAGS=-mod=mod `
GINKGO = go run github.com/onsi/ginkgo/v2/ginkgo@v2.1.4

And then use it for example like 
```
vet:
	$(GO_MOD_FLAGS) go vet ./...

test-e2e-upgrade: manifests
	KUBECONFIG=$(KUBECONFIG) OPERATOR_NAMESPACE=$(OPERATOR_NAMESPACE) $(GO_MOD_FLAGS) GINKGO="$(GINKGO)" ./hack/run-e2e-test-upgrade.sh $(e2e_test_args) $(E2E_TEST_SUITE_ARGS)
```

Because currently, for test-e2e-upgrade target, the GOFLAGS=-mod=mod is interpreted as command name

Reply:
Used different approach